tests/appsec/test_runtime_activation.py

```diff
@@ -12,6 +12,9 @@
 @scenario("APPSEC_RUNTIME_ACTIVATION")
 @released(java="0.115.0", cpp="?", dotnet="2.16.0", php="?", python="?", ruby="?", nodejs="?", golang="?")
 @irrelevant(context.appsec_rules_file == "")
+@irrelevant(
+    context.library >= "java@1.1.0" and context.appsec_rules_file is not None, reason="Can't test with cutom rule file"
```

Comment:
Fwiw, I don't think this is specific to any library version.

Reply:
Yep, I'll skip them whem they will start failiong ...